src/helpers.js

```diff
@@ -44,28 +46,33 @@ export async function enablePackage(packageName: string): Promise<void> {
   }
 }
 
-export function getDependencies(packageName: string): Array<Dependency> {
+const DEPENDENCY_REGEX = /^([^#:]+)(#[^:]+)?(:.+)?$/
+export async function getDependencies(packageName: string): Promise<Array<Dependency>> {
   const toReturn = []
   const packageModule = atom.packages.getLoadedPackage(packageName)
   const packageDependencies = packageModule && packageModule.metadata['package-deps']
 
   if (packageDependencies) {
     for (const entry of (packageDependencies: Array<string>)) {
-      let entryName = entry
-      let entryUrl = entry
-
-      if (entry.indexOf('#') > -1) {
-        [entryName, entryUrl] = entry.split('#')
-      }
-
-      if (__steelbrain_package_deps.has(entryName) || atom.packages.resolvePackagePath(entryName)) {
+      const matches = DEPENDENCY_REGEX.exec(entry)
+      if (matches === null) {
+        console.error('[Package-Deps] Error parsing package-deps entry of', packageName, 'with value:', entry)
```

Comment:
You already have the message tagged with `[Package-Deps]`, no need to repeat it in the message as well.